### PR TITLE
Updated DB Dockerfile to Postgres 13.3

### PR DESCRIPTION
**Pull request by wlad**
updated base image to postgres:13.3-alpine

db setup is fully based on [cloud-db-setup.sql](https://github.com/ehrbase/ehrbase/blob/develop/base/db-setup/cloud-db-setup.sql), no extension required any more

removed extensions:
- temporal_tables
- jsquerry

moved old files into archive folder (just in case)

### CODE REVIEW

File: archive/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# docker
```
Comment (jakesmolka) [inline archive/README.md:1]: What's your idea behind keeping this as an archived file? For the normal use cases the git history is enough I think.

(Same for all files moved to "archive".)

**Review by jakesmolka — COMMENTED** [review]
See my inline comment. Otherwise, LGTM!

**Comment by wlad** [comment]
Before I push `latest` tag to DockerHub I'd like to see it tested and get some feedback here if there are any issues.

**Comment by jakesmolka** [comment]
I've been using this image and there weren't any problems.

**Pull request merged**